crates/bevy_window/src/window.rs

```diff
@@ -658,6 +685,12 @@ pub struct InternalWindowState {
     maximize_request: Option<bool>,
     /// Unscaled cursor position.
     physical_cursor_position: Option<DVec2>,
+    /// The window's current theme
+    ///
+    /// ## Platform-specific
+    ///
+    /// - iOS / Android / Web / x11: Unsupported.
```

Comment:
Documenting internal fields, I'd suggest talking as if the reader was a contributor.

```suggestion
    ///
    /// This is ignored on iOS, Android and Web platforms.
```